--- check-pw.x.j	2016-02-18 19:50:09.000000000 -0800
+++ patch-files/check-pw.x.j	2016-02-18 19:51:45.000000000 -0800
@@ -2,7 +2,7 @@
 
 # Automated checks for pw.x - PG 2007-2014
 #
-. ../../environment_variables
+. ROLLPREFIX/tests/environment_variables
 #
 # You shouldn't need to modify anything below this line.
 #
@@ -38,8 +38,8 @@
 # taken from examples - not sure it is really needed
 if test "`echo -e`" = "-e" ; then ECHO=echo ; else ECHO="echo -e" ; fi
 
-ESPRESSO_ROOT=`cd ../../ ; pwd`
-ESPRESSO_TMPDIR=$ESPRESSO_ROOT/tempdir/
+ESPRESSO_ROOT=ROLLPREFIX
+ESPRESSO_TMPDIR=tempdir
 ESPRESSO_PSEUDO=$ESPRESSO_ROOT/pseudo/
 
 # no need to specify outdir and pseudo_dir in all *.in files
@@ -256,7 +256,7 @@
   # run the code in the scratch directory
   #
   cd $ESPRESSO_TMPDIR
-  $PARA_PREFIX $ESPRESSO_ROOT/PW/src/pw.x $plugin_name $PARA_POSTFIX \
+  $PARA_PREFIX $ESPRESSO_ROOT/bin/pw.x $PARA_POSTFIX \
         -i $TESTDIR/$name.in > $TESTDIR/$name.out
   if test $? != 0; then
      $ECHO "FAILED with error condition!"
@@ -290,7 +290,7 @@
      # run the code in the scratch directory
      #
      cd $ESPRESSO_TMPDIR
-     $PARA_PREFIX $ESPRESSO_ROOT/PW/src/pw.x $PARA_POSTFIX \
+     $PARA_PREFIX $ESPRESSO_ROOT/bin/pw.x $PARA_POSTFIX \
              -i $TESTDIR/$name.in$n > $TESTDIR/$name.out$n
      if test $? != 0; then
         $ECHO "FAILED with error condition!"
