--- qe-6.3/environment_variables	2018-07-06 08:36:48.000000000 -0700
+++ patch-files/environment_variables	2018-07-16 18:33:55.661415475 -0700
@@ -17,11 +17,11 @@
 
 # The following should be good for most cases 
 
-PREFIX=`cd ../../.. ; pwd`
+PREFIX=ROLLPREFIX
 BIN_DIR=$PREFIX/bin
 PSEUDO_DIR=$PREFIX/pseudo
 # Beware: everything in $TMP_DIR will be destroyed !
-TMP_DIR=$PREFIX/tempdir
+TMP_DIR=tempdir
 
 # There should be no need to change anything below this line
 
@@ -56,7 +56,7 @@
 # a small number of processors; do not use tests and examples to benchmark
 # parallelism, do not run on too many processors
 
-PARA_PREFIX=" "
+PARA_PREFIX="mpirun RUNASROOT -np 4"
 PARA_PREFIX="mpirun -np 4"
 #
 # available flags: 
